packs/feats/storming-gaze.json

```diff
@@ -24,7 +24,53 @@
             "remaster": true,
             "title": "Pathfinder Lost Omens: Tian Xia Character Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "alwaysActive": true,
+                "key": "RollOption",
+                "mergeable": true,
+                "option": "storming-gaze",
+                "suboptions": [
+                    {
+                        "label": "PF2E.TraitElectricity",
+                        "value": "electricity"
+                    },
+                    {
+                        "label": "PF2E.TraitSonic",
+                        "value": "sonic"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "itemId": "{item|id}",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "not": "storming-gaze:tempest-gaze"
+                    }
+                ],
+                "property": "traits",
+                "value": "{item|flags.pf2e.rulesSelections.stormingGaze}"
+            },
+            {
+                "itemId": "{item|id}",
+                "key": "ItemAlteration",
+                "mode": "override",
+                "predicate": [
+                    {
+                        "not": "storming-gaze:tempest-gaze"
+                    }
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Oni.StormingGaze.Base"
```

Comment:
Why override the description here?

Reply:
The idea is to use the suboptions to switch between electricity and sonic, and adding the traits as appropriate and overriding the description with the resolved damage type. Alternatively an actor flag could be used, or just not doing it and keeping the two damage buttons.

Reply:
Since it's on the same item you can reference flags just fine without item alterations. `@Damage` is fancy like that

Reply:
See Blasting Beams, for example

Reply:
Anyways if it's necessary to use actor flags I prefer that to shoving more into localization files